sources/packages/backend/apps/api/src/constants/error-code.constants.ts

```diff
@@ -240,6 +240,11 @@ export const APPLICATION_RESTRICTION_BYPASS_NOT_FOUND =
 export const APPLICATION_RESTRICTION_BYPASS_IS_NOT_ACTIVE =
   "APPLICATION_RESTRICTION_BYPASS_IS_NOT_ACTIVE";
 
+/**
+ * Current application not found.
+ */
+export const CURRENT_APPLICATION_NOT_FOUND = "CURRENT_APPLICATION_NOT_FOUND";
```

Comment:
We can throw HTTP errors from controller service and custom error is not required.